fuzz/fuzz_targets/decompress_lzma.rs

```diff
@@ -0,0 +1,18 @@
+#![no_main]
+#[macro_use]
+extern crate libfuzzer_sys;
+extern crate lzma_rs;
+
+use lzma_rs::error::Result;
+
+fn decode_lzma(compressed: &[u8]) -> Result<Vec<u8>> {
+    let mut bf = std::io::Cursor::new(compressed);
+
+    let mut decomp: Vec<u8> = Vec::new();
+    lzma_rs::lzma_decompress(&mut bf, &mut decomp)?;
+    Ok(decomp)
+}
+
+fuzz_target!(|data: &[u8]| {
+    let _decomp = decode_lzma(data);
```

Comment:
When writing benchmarks, I recall the pattern of having a lambda function returning a value of some arbitrary type `T`. Then, the benchmark framework would mark this value as "consumed" to avoid optimizing it away.

Although the goals are different, could a similar pattern be used here? Or does `fuzz_target` require a function returning `()`?

```rust
fuzz_target!(|data: &[u8]| {
    decode_lzma(data)
}
```

Reply:
Yes, returning `()` is required by the function signature.